sw/device/rom_exts/manifest.h

```diff
@@ -0,0 +1,220 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_ROM_EXTS_MANIFEST_H_
+#define OPENTITAN_SW_DEVICE_ROM_EXTS_MANIFEST_H_
+
+// This file is the "ROM_EXT manifest header template".
+// It has been generated by running the command:
+// $ util/rom-ext-manifest-generator.py.
+
+// Header Extern Guard (so header can be used from C and C++)
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * ROM_EXT manifest field manifest_identifier offset from the base.
+ */
+#define ROM_EXT_MANIFEST_IDENTIFIER_OFFSET 0u
```

Comment:
Is there a particular reason these are #defines as opposed to being fields in an anonymous enum? I'm guessing this is so that this is usable from assembly? If so, we might want to include a comment noting that.

Reply:
Our pervasive use of the `u` literal suffix prevents us using this from assembly.

I think we started with defines because that matches almost everywhere else we generate information like this.

Reply:
And there are already existing routines for that in `c.py`